src/MLIAP/mliap_descriptor_so3.cpp

```diff
@@ -0,0 +1,298 @@
+/* ----------------------------------------------------------------------
```

Comment:
If you want that the LAMMPS clang-format formatting is applied, then please reformat your code with it, otherwise add a `// clang-format off` as the first line.